lib/price.rb

```diff
@@ -0,0 +1,24 @@
+class Price
```

Comment:
Would it be possible to simplify this by dividing the pence amounts by a float (`pence / 100.0`) and then using `stprintf` to add the required number of trailing / preceding zeros?

Reply:
Yep, that would be a better solution.